src/LaurentMPoly.jl

```diff
@@ -143,24 +143,14 @@ end
 ###############################################################################
 
 @doc raw"""
-    LaurentPolynomialRing(R::Ring, s::Vector{T}; cached::Bool = true) where T <: VarName
+    LaurentPolynomialRing(R::AbstractAlgebra.Ring, s...; cached::Bool = true)
 
-Given a base ring `R` and an array of strings `s` specifying how the
+Given a base ring `R` and variable names `s` specifying how the
```

Comment:
That was my plan indeed. Currently, this is all in the `variable_names` docstring. Though, as users would usually not call `variable_names` directly, I will move most of the docstring into its own section in the manual.

Reply:
OK.

Just to make sure we don't have a misunderstanding, I'll be explicit here: While a full explanation for users (and perhaps another explanation for *developers*) should be in the manual, I think it would be good to have at least some information in some docstring. But of course that does not have to be a single docstring everything points to: if we e.g. have an examples section with a few non-trivial examples in the primary docstring of `polynomial_ring`, `free_group`, `free_associative_algebra`, etc. etc. plus a note "For further options to specify variable names, see LINK-TO-MANUAL" should be OK, too.